File: server/src/main/java/com/quotegarden/quote/domain/Quote.java
package com.quotegarden.quote.domain;

import jakarta.persistence.*;

@Entity
@Table(name = "quote")
public class Quote {
    @Id public String id;
    public String content;
    public String translation;
    public String author;
    public String category;
    public Integer difficulty;
    public String audioUrl;
    public String imageUrl;
    public Long updatedAt;
    public Long deletedAt;
}

File: app/src/main/java/com/dailymind/core/model/Quote.kt
package com.dailymind.core.model

data class Quote(
    val id: String,
    val content: String,
    val translation: String?,
    val author: String?,
    val category: String?,
    val difficulty: Int?,
    val audioUrl: String?,
    val imageUrl: String?,
    val updatedAt: Long
)
